spec/forms/steps/lateness/lateness_reason_form_spec.rb

```diff
@@ -5,11 +5,38 @@
     tribunal_case:   tribunal_case,
     lateness_reason: lateness_reason
   } }
-  let(:tribunal_case)   { instance_double(TribunalCase, lateness_reason: nil) }
+  let(:tribunal_case)   { instance_double(TribunalCase, lateness_reason: nil, in_time: in_time) }
   let(:lateness_reason) { nil }
+  let(:in_time)         { nil }
 
   subject { described_class.new(arguments) }
 
+  describe '#lateness_unknown?' do
+    context 'when appeal is late' do
```

Comment:
Aren't 'when appeal is late' and 'when appeal is not late' the wrong way round, @zheileman?